test/libyul/yulOptimizerTests/fullSuite/aztec.yul

```diff
@@ -238,8 +238,8 @@
 //         let notes := add(0x04, calldataload(0x04))
 //         let m := calldataload(0x24)
 //         let n := calldataload(notes)
-//         let gen_order := 0x30644e72e131a029b85045b68181585d2833e84879b9709143e1f593f0000001
-//         let challenge := mod(calldataload(0x44), gen_order)
+//         let _1 := 0x30644e72e131a029b85045b68181585d2833e84879b9709143e1f593f0000001
```

Comment:
Would be nice to keep the name (i.e. pass it as hint somewhere), but not too important I guess...

Reply:
I know, but we can't this way.